orc8r/cloud/go/services/orchestrator/obsidian/handlers/network_handlers.go

```diff
@@ -53,6 +53,22 @@ func registerNetwork(c echo.Context) error {
 	if err != nil {
 		return echo.NewHTTPError(http.StatusBadRequest, err)
 	}
+
+	// Creating default tier for each created network
+	for _, s := range createdNetworks {
+		defaultTier := &models.Tier{
+			ID:       "default",
+			Gateways: models.TierGateways{},
+			Images:   models.TierImages{},
+			Version:  models.TierVersion("1.0"),
+		}
+		entity := defaultTier.ToNetworkEntity()
+		_, err := configurator.CreateEntity(c.Request().Context(), s.ID, entity, serdes.Entity)
+		if err != nil {
+			// Handling error: this may not block the register workflow, instead it prints error logs
```

Comment:
I'd say let's use a multierror for this, and *do* throw an err if any of the default tiers fail to create

E.g. https://github.com/magma/magma/blob/ff012313835a8ecb0cd4b9ac2055186468295838/lte/cloud/go/services/subscriberdb_cache/config.go#L29-L36